assets/stylesheets/base/_base.scss

```diff
@@ -1,18 +1,20 @@
 html {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 100%;
+  font-family: family('body');
```

Comment:
I still prefer this because it makes the math easier, but USWDS 2.0 doesn't seem to play as nicely with it.
```
html {
  font-size: 10px;
}

body {
  font-size: 1.7rem;
}
```